### PR TITLE
Changed Luke Scattering rate in G4CMPProcessUtils

### PR DESCRIPTION
**Pull request by emichaud4**
In G4CMPProcessUtils, multiplied line 472 by mach to get the correct Luke scattering rate.

### CODE REVIEW

**Comment by kelseymh** [comment]
Emile, we *never* merge anything directly onto master/main.  We are following the GitFlow workflow model:

1. Make feature branches from *develop* (or other feature branches).
2. Merge feature branches back onto *develop*, or onto the parent feature branch, and apply a tag with the branch/ticket/issue name.
3. Merge ONLY develop onto *master*/*main*, and apply a semantic versioning release tag.

**Comment by kelseymh** [comment]
At the top of `G4CMPProcessUtils.cc`, add a short (one line or 1-1/2 line) comment with a date stamp, in chronological order with the existing comments.  Describe what you are changing and why.

File: library/src/G4CMPProcessUtils.cc [hunk 1]
```diff
@@ -504,5 +504,5 @@ G4CMPProcessUtils::ChargeCarrierTimeStep(G4double mach, G4double l0) const {
   const G4double velLong = theLattice->GetSoundSpeed();
 
   const G4double tstep = 3.*l0/velLong;
-  return (mach<1.) ? tstep : tstep*mach/((mach-1)*(mach-1)*(mach-1));
+  return (mach<1.) ? tstep : tstep*mach*mach/((mach-1)*(mach-1)*(mach-1));
```
Comment (kelseymh) [inline library/src/G4CMPProcessUtils.cc:507]: Add a comment showing the derivation that gets the correct "extra" factor of mach in the numerator.

Comment (emichaud4) [reply library/src/G4CMPProcessUtils.cc]: it's done!

**Comment by emichaud4** [comment]
> At the top of `G4CMPProcessUtils.cc`, add a short (one line or 1-1/2 line) comment with a date stamp, in chronological order with the existing comments. Describe what you are changing and why.

Also done!

**Comment by emichaud4** [comment]
> Emile, we _never_ merge anything directly onto master/main. We are following the GitFlow workflow model:
> 
> 1. Make feature branches from _develop_ (or other feature branches).
> 2. Merge feature branches back onto _develop_, or onto the parent feature branch, and apply a tag with the branch/ticket/issue name.
> 3. Merge ONLY develop onto _master_/_main_, and apply a semantic versioning release tag.

Sorry, I though I was asking to merge into develop, I should have double checked, my bad!

**Pull request merged**